radlab-ui/webapp/src/components/outputs/ModuleOutputs.tsx

```diff
@@ -19,8 +20,8 @@ interface ModuleOutputsProps {
 
 const isURL = (val: string) => val.search(/^https?:\/\//) > -1
 
-const renderTFValue = (val: string): JSX.Element => {
-  if (isURL(val)) {
+const renderTFValue = (val: any): JSX.Element => {
```

Comment:
typically better to use `unknown` instead of `any`. The former forces you to check types, where the latter turns off type checking.

Reply:
Changes will be made in a separate PR based on your feedback.